pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -93,7 +95,8 @@ func New(namespace string, secretsLister internalinformers.SecretLister, issuer
 		return nil, err
 	}
 
-	vcertClient, err := vcert.NewClient(cfg)
+	// Don't authenticate just now, do this later in VerifyCredentials
```

Comment:
Explain "why" it is important to not authenticate here. 
Explain that, by default, vcert.NewClient will use apikey authentication, if we supply a username / password,
and that we want to avoid that.

Reply:
I hope that the new comments adequately describe this change.